src/product.js

```diff
@@ -142,7 +142,7 @@ const getApplicationPath = (product) => {
     return oldMatch[1];
   }
 
-  throw new Error(`Can't find application path for ${bin}`);
+  throw new Error(`Can't find application path for ${binContent}`);
```

Comment:
I don't get this point. Can you explain the reason behind this change?

Reply:
The problem for me here was exactly the one you see in the exception. The variable `bin` is not defined in this scope. So JS will throw the ReferenceError exception.

Reply:
ok, my bad I didn't read my code clearly 😄  

In that case I prefer to put the `product.key` because the `binContent` can be big for non Toolbox case

Reply:
Done that, I also think it is better. I just put binContent because it was the closest to bin. hehehe